include/matx/operators/inverse.h

```diff
@@ -107,6 +107,24 @@ namespace detail {
   };
 }
 
+/**
+ * Performs a matrix inverse on a square matrix. The inverse API currently uses
+ * cuBLAS as a backend and uses `getri/getrf` functions for LU decomposition.
```

Comment:
This comment may be out of date with a recent commit by @tbensonatl , where it uses another function in certain cases: https://github.com/NVIDIA/MatX/commit/d9053d6c371b60aab70b6f37d330bd1ada6b8697

Reply:
Updated the comment